src/main/java/com/cloudbees/jenkins/support/impl/AboutJenkins.java

```diff
@@ -643,6 +644,13 @@ private String getLabelString(Node n) {
                     Markdown.escapeBacktick(ContentFilter.filter(filter, jenkins.getRootDir().getAbsolutePath())) + "`");
             out.println("      - Labels:         " + ContentFilter.filter(filter, getLabelString(jenkins)));
             out.println("      - Usage:          `" + jenkins.getMode() + "`");
+            Optional.ofNullable(jenkins.toComputer()).ifPresent(computer ->
+                out.println("      - Marked Offline: " + computer.isTemporarilyOffline()));
+            if(jenkins.getChannel() == null) {
```

Comment:
Added also the `- Status` for the built-in node to bring parity with other nodes. Not sure if it is possible to have a `null` channel for the built-in node ? unless maybe a plugin or a script does this explicitly.